templates/glance/config/glance-api.conf

```diff
@@ -8,7 +8,7 @@ bind_host=0.0.0.0
 bind_port=9292
 workers=3
 image_cache_dir=/var/lib/glance/image-cache
-log_config_append=/etc/glance/logging.conf
+log_config_append=/etc/glance/glance.conf.default.d/logging.conf
 enabled_backends=default_backend:file
```

Comment:
I think similar path change is required at line 38 and 41

Reply:
Right, thanks!

Reply:
did it for policy, I'll do it later for the other files